guides/integration-release.md

```diff
@@ -39,3 +39,20 @@ Also, when writing the final changelogs, you might notice some inconsistencies,
 > - At the top right part of the screen click on `Re-run all jobs`.
 
 **This keeps consistency between the PRs history and the release changelogs**.
+
+### What to do when Failing a Release 
+
+In some repositories, [GitHub actions are triggered](https://github.com/meilisearch/meilisearch-js/blob/main/.github/workflows/publish.yml) after a tag creation following the release publication. It might happen that one of these actions failed because pre-requisites are not met (i.e., the new version in the correct file). Don't worry; it is fixable! These are the steps: 
```

Comment:
```suggestion
In some repositories, [GitHub Actions are triggered](https://github.com/meilisearch/meilisearch-js/blob/main/.github/workflows/publish.yml) after a tag creation following the release publication. It might happen that one of these actions failed because pre-requisites are not met (i.e., the new version in the correct file). Don't worry; it is fixable! These are the steps: 
```